specification/dfp/resource-manager/Microsoft.DFP/preview/2021-02-01-preview/dfp.json

```diff
@@ -0,0 +1,782 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-02-01-preview",
+    "title": "DFP",
+    "description": "DFP Web API provides a RESTful set of web services that enables users to create, retrieve, update, and delete DFP instances",
+    "x-ms-code-generation-settings": {
+      "name": "DFPManagementClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Dfp/instances/{instanceName}": {
```

Comment:
its not clear what Dfp stands for .. if this is not an industry standard acronym please consider using the full name

Reply:
Renamed to Microsoft.Dynamics365Fraudprotection